src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/client/prisonersearch/PrisonerSearchApiClient.kt

```diff
@@ -62,4 +62,27 @@ class PrisonerSearchApiClient(
     } catch (e: Exception) {
       throw PrisonerSearchApiException("Error retrieving prisoner by prisonNumber $prisonNumber", e)
     }
+
+  /**
+   * Returns a list of prisoners from prisoner-search-api, identified by their prisonNumber
+   */
+  fun getPrisonersIn(prisonNumbers: List<String>): List<Prisoner> =
```

Comment:
Need this for efficiency to save spamming the prisoner-search API (thank goodness they already had an endpoint that support this! )